.travis.yml

```diff
@@ -1,42 +0,0 @@
-language: go
-
-# The version used here should match the BUILD_IMAGE variable in the
-# Makefile.
-go:
-  - "1.15.3"
-
-# Enable building in Travis using forked repos.
-go_import_path: sigs.k8s.io/kubefed
-
-# Request latest Travis distro for systemd requirement.
-dist: xenial
-sudo: required
-
-services:
-  - docker
-
-before_install:
-- bash ./hack/verify-docfiles.sh && DOC_ONLY=true && echo "Only doc files have changed!" || test true
-
-script:
-  - test $DOC_ONLY && echo "Skipped!" || DOWNLOAD_BINARIES=y bash -x ./scripts/pre-commit.sh
-
-# Note on configuration of quay account:
-#
-# The image push is done using a quay robot account configured in the
-# kubernetes-multicluster org.
-#
-# There are two environment variables configured in travis that allow the travis
-# build job to use this robot account:
-#
-# - QUAY_USERNAME
-# - QUAY_PASSWORD
-#
-# These are statically configured in the travis settings and are set by travis
-# in the build job's execution environment.
-after_success:
-  - test $DOC_ONLY && echo "Skipped!" || make push
```

Comment:
This path isn't reflect in the GitHub Actions workflow above. Given the Makefile reads the TRAVIS_PULL_REQUEST environment variable in the push target, I take it we need to rework that a bit. My preference would be to just push anytime the target is called and only call that `on: push.tags`. wdyt?

Reply:
Done in 77a49debaedde724862dbe6157c60b982664c93a (i hope!)